internal/server/event/conversion/conversions.go

```diff
@@ -494,39 +438,12 @@ func FromInternalJobRunPreempted(queueName string, jobSetName string, time time.
 		return nil, nil
 	}
 
-	jobId, err := armadaevents.UlidStringFromProtoUuid(e.PreemptedJobId)
-	if err != nil {
-		return nil, err
-	}
-	runId, err := armadaevents.UuidStringFromProtoUuid(e.PreemptedRunId)
-	if err != nil {
-		return nil, err
-	}
-
-	preemptiveJobId := ""
```

Comment:
we don't set these  anywhere.  We should remove these from the event but we can do that in a separate PR